front-end/src/common/PhotoGrid/PhotoGrid.tsx

```diff
@@ -15,6 +21,32 @@ const localeDateOptions: Intl.DateTimeFormatOptions = {
 };
 
 export const PhotoGrid: React.FC<PhotoGridProps> = (props) => {
+  const [photoCarouselSrc, setPhotoCarouselSrc] = React.useState("");
+  const [showPhotoCarousel, setShowPhotoCarousel] = React.useState(false);
+
+  const onEscapePhotoCarousel = React.useCallback((ev: KeyboardEvent) => {
+    console.log("hehe");
```

Comment:
I think we can remove this `console.log()`

Reply:
lol, good catch